features/step_definitions/news_article_steps.rb

```diff
@@ -117,6 +117,8 @@
   select "", from: "edition_lead_organisation_ids_1"
 
   click_button "Save"
+  # TODO  investigate why save legacy breaks this
```

Comment:
Is this still something to do?

Reply:
Yes it is, it was days ago and I forgot about it. The functionality works when you try it manually, something in the test setup. I'll put it in the card